pom.xml

```diff
@@ -11,6 +11,10 @@
     <properties>
         <maven.compiler.source>14</maven.compiler.source>
         <maven.compiler.target>14</maven.compiler.target>
+
+        <ld.library.path>./lib</ld.library.path>
```

Comment:
`LD_LIBRARY_PATH` is not a Maven property, it's an environment variable. Why are you setting it here?